### PR TITLE
[IMP] website: rename 'choose a theme' to 'pick a theme'

### PR DESCRIPTION
**Pull request by JKE-be**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
                         <div class="row mt16 o_settings_container" id="website_settings" attrs="{'invisible': [('website_id', '=', False)]}">
                             <div class="col-12 o_setting_box" id="website_action_setting" style="margin-left: 30px; margin-bottom: 16px;">
                                 <button name="website_go_to" type="object" string="Go to Website" class="btn btn-primary" icon="fa-globe"/>
-                                <button name="install_theme_on_current_website" type="object" string="Choose a theme" class="ml-2 btn btn-primary" icon="fa-paint-brush"/>
+                                <button name="install_theme_on_current_website" type="object" string="Pick a theme" class="ml-2 btn btn-primary" icon="fa-paint-brush"/>
```
Comment (sbuhl) [inline addons/website/views/res_config_settings_views.xml:43]: Pick a Theme

File: addons/website/views/website_views.xml [hunk 2]
```diff
@@ -76,7 +76,7 @@
             <field name="arch" type="xml">
                 <xpath expr="//form" position="inside">
                     <footer>
-                        <button name="create_and_redirect_to_theme" type="object" string="Choose a theme" class="btn btn-primary" icon="fa-paint-brush"/>
+                        <button name="create_and_redirect_to_theme" type="object" string="Pick a theme" class="btn btn-primary" icon="fa-paint-brush"/>
```
Comment (sbuhl) [inline addons/website/views/website_views.xml:79]: Pick a Theme

File: addons/website/views/website_views.xml [hunk 3]
```diff
@@ -420,7 +420,7 @@
         </record>
 
         <record id="theme_install_kanban_action" model="ir.actions.act_window">
-            <field name="name">Choose a theme for your website</field>
+            <field name="name">Pick a theme</field>
```
Comment (sbuhl) [inline addons/website/views/website_views.xml:423]: Pick a Theme.

**Review by sbuhl — COMMENTED** [review]
Thank you!

**Comment by JKE-be** [comment]
@robodoo r+

**Pull request closed without merging**